docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,113 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables ({issue}`24447`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from Delta Lake tables by compressing files
+  with `ZSTD` by default, instead of the previous `SNAPPY`.  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
+* Replace predicate pushdown with `min`, `max`, and `options` column properties. ({issue}`24147`)
```

Comment:
I think we probaly want to just remove the mention of predicate pushdown and mention addition of the column properties and what they add.

Reply:
It's breaking change, so we should definitely say that predicate pushdown are not supported anymore.